torch/csrc/jit/serialization/python_print.cpp

```diff
@@ -1075,10 +1075,10 @@ struct PythonPrintImpl {
 
         if (auto selfClass = self->type()->cast<ClassType>()) {
           deps_table_.add(selfClass);
-          const Function& method = selfClass->getMethod(node->s(attr::name));
-          TORCH_INTERNAL_ASSERT(
-              method.qualname() ==
-              QualifiedName(selfClass->name()->qualifiedName(), methodName));
+          // const Function& method = selfClass->getMethod(node->s(attr::name));
+          // TORCH_INTERNAL_ASSERT(
+          //     method.qualname() ==
+          //     QualifiedName(selfClass->name()->qualifiedName(), methodName));
```

Comment:
Is this intended?

Reply:
This is changed to reflect how we serialize overloaded method